filter/series_by_tag.go

```diff
@@ -34,13 +37,15 @@ type TagSpec struct {
 	Value    string
 }
 
-func transformWildcardToRegexpInSeriesByTag(input string) string {
-	var result = input
-	var re = regexp.MustCompile(`\{(.*?)\}`)
-	var correctLengthOfMatchedWildcardIndexesSlice = 4
+func transformWildcardToRegexpInSeriesByTag(input string) (string, bool) {
+	var (
+		result                                     = input
+		correctLengthOfMatchedWildcardIndexesSlice = 4
```

Comment:
Maybe we sould make it a constant and also use in `ParseSeriesByTag`, where magic number 4 is used